code/theme-default.css

```diff
@@ -0,0 +1,127 @@
+   body {
```

Comment:
you are using formatting for css? do it  in ATOM somehow

Reply:
I have no clue what this means, please be more specific here

Reply:
Format of file. I don't know why you have indent of each row. 
body {
  xxx:xxx;
}

no this 
  body {
    xxx>xxx;
  ;